proj/src/main/webapp/static/chat.html

```diff
@@ -2,50 +2,43 @@
 <html lang="en" dir="ltr">
   <head>
     <meta charset="utf-8">
-    <title>Welcome</title>
-    <!-- Bootstrap: Latest compiled and minified CSS -->
+    <title>Chat Room</title>
+
+    <!-- Bootstrap Scripts -->
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
-    <!-- Bootstrap: jQuery library -->
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
-    <!-- Bootstrap: Latest compiled JavaScript -->
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
-    <link rel="stylesheet" href="../style.css">
-    <!-- Font  -->
-    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap" rel="stylesheet">
+    
 
+    <!-- Firebase Scripts -->
     <!-- The core Firebase JS SDK is always required and must be listed first -->
     <script src="https://www.gstatic.com/firebasejs/7.15.4/firebase-app.js"></script>
-
-    <!-- TODO: Add SDKs for Firebase products that you want to use
-        https://firebase.google.com/docs/web/setup#available-libraries -->
     <script src="https://www.gstatic.com/firebasejs/7.15.4/firebase-analytics.js"></script>
-
-    <script src="../app.js"></script>
-
     <script src="https://www.gstatic.com/firebasejs/7.15.4/firebase-auth.js"></script>
     <script src="https://www.gstatic.com/firebasejs/7.15.4/firebase-database.js"></script>
     <script src="https://www.gstatic.com/firebasejs/7.15.4/firebase-messaging.js"></script>
-  </head>
-  <body>
-      <div class="container screen chat-page">
+    <script src="../app.js"></script>
+
+    <!-- Font  -->
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="../style.css">
+    </head>
+    
+    <body onload="init()">
+        <div class="container screen chat-page">
             <div class="row">
             <div class="col" id="chat-screen"> 
                 <h1> Chat Room </h1>
                 <div class="chat-and-input"> 
                     <div class="converstation">
-                        <ul id="chat-messages">
-                        <li>Coffee</li>
-                        <li>Tea</li>
-                        <li>Milk</li>
-                        </ul>
+                        <ul id="chat-as-list">
```

Comment:
Does this need a closing tag? Also is it possible to include a username with the message? Would this depend on things that are not yet ready? If so, we can save that addition for later.

Reply:
Arrington actually addressed that in another pull request.  The list is changed into a div.

Reply:
Depending on who is closer to submitting their code first, you might want to pull that change into this branch to handle merge conflicts ahead of time.

Reply:
good idea; i can wait for her to merge with master and i will pull from there

Reply:
If this one is ready first, you can also just merge this and then work with her to resolve merge conflicts on her PR as long as you both are in agreement, either is fine.

Reply:
gotcha